# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1382907950 -3600
#      Sun Oct 27 22:05:50 2013 +0100
# Node ID 8402768afd2dd207a4b8a6aa8cbc0ffae55d4a8a
# Parent  016634fb3de73c93ce95bda26a1fc965ed012163
updated language files (with bash lexer editor styles) and finished de_DE

diff --git a/libgui/languages/be_BY.ts b/libgui/languages/be_BY.ts
--- a/libgui/languages/be_BY.ts
+++ b/libgui/languages/be_BY.ts
@@ -84,16 +84,89 @@
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1527"/>
         <source>copied selection to clipboard</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
+    <name>QsciLexerBash</name>
+    <message>
+        <location filename="../../../qsci/qscilexerbash.cpp" line="+208"/>
+        <source>Default</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Error</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Comment</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Number</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Keyword</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Double-quoted string</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Single-quoted string</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Operator</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Identifier</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Scalar</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Parameter expansion</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Backticks</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Here document delimiter</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Single-quoted here document</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
     <name>QsciLexerBatch</name>
     <message>
         <location filename="../../../qsci/qscilexerbatch.cpp" line="+179"/>
         <source>Default</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1498,22 +1571,22 @@ has been deleted or renamed. Do you want
         <location line="+1"/>
         <source>Create script</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation type="unfinished">Загрузіць прастору зменных</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation type="unfinished">Пра Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation type="unfinished">&amp;Файл</translation>
@@ -1569,23 +1642,33 @@ has been deleted or renamed. Do you want
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation type="unfinished"></translation>
     </message>
diff --git a/libgui/languages/de_DE.ts b/libgui/languages/de_DE.ts
--- a/libgui/languages/de_DE.ts
+++ b/libgui/languages/de_DE.ts
@@ -925,22 +925,22 @@ zu: </translation>
         <location line="+1"/>
         <source>Create script</source>
         <translation>Skript erzeugen</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation>Lade Arbeitsumgebung</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation>Über Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation>&amp;Datei</translation>
@@ -996,23 +996,33 @@ zu: </translation>
         <translation>Kopieren</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation>Einfügen</translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation>Arbeitsumgebung speichern als</translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation>Die Datei &apos;%1&apos; mit Versionshinweisen ist leer.</translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation>Die Datei &apos;%1&apos; mit Versionshinweisen kann nicht gelesen werden.</translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Versionshinweise zu Octave</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Arbeitsverzeichnis setzen</translation>
     </message>
diff --git a/libgui/languages/en_US.ts b/libgui/languages/en_US.ts
--- a/libgui/languages/en_US.ts
+++ b/libgui/languages/en_US.ts
@@ -905,22 +905,22 @@ has been deleted or renamed. Do you want
         <location line="+1"/>
         <source>Create script</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation type="unfinished"></translation>
@@ -976,23 +976,33 @@ has been deleted or renamed. Do you want
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation type="unfinished"></translation>
     </message>
diff --git a/libgui/languages/es_ES.ts b/libgui/languages/es_ES.ts
--- a/libgui/languages/es_ES.ts
+++ b/libgui/languages/es_ES.ts
@@ -908,22 +908,22 @@ has been deleted or renamed. Do you want
         <source>Create script</source>
         <translatorcomment>Ha sido utilizada la traducción &quot;guión&quot; de script: http://es.wikipedia.org/wiki/Script#Traducci.C3.B3n</translatorcomment>
         <translation>Crear un guión</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation>Cargar espacio de trabajo</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation>Acerca de Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation>&amp;Archivo</translation>
@@ -980,23 +980,33 @@ has been deleted or renamed. Do you want
         <translation>Copiar</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation>Pegar</translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation>Guardar espacio de trabajo como</translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Notas de la versión de Octave</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Fijar directorio de trabajo</translation>
     </message>
diff --git a/libgui/languages/fr_FR.ts b/libgui/languages/fr_FR.ts
--- a/libgui/languages/fr_FR.ts
+++ b/libgui/languages/fr_FR.ts
@@ -928,22 +928,22 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <location line="+1"/>
         <source>Create script</source>
         <translation>Créer un fichier de script</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation>Charger l&apos;espace de travail</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation>À propos d&apos;Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation>&amp;Fichier</translation>
@@ -999,23 +999,33 @@ a été supprimé ou rénommé. Voulez-vous l&apos;enregistrer maintenant ?%2</translation>
         <translation>Copier</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation>Coller</translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation>Enregistrer l&apos;espace de travail sous</translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Notes de la version d&apos;Octave</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Définir le répertoire de travail</translation>
     </message>
@@ -1721,23 +1731,23 @@ a:link { text-decoration: underline; col
 &lt;li&gt;Open the documentation browser of Octave GUI with the help menu&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;</source>
         <translation>&lt;html&gt;&lt;head&gt;
 &lt;style&gt;
 a:link { text-decoration: underline; color: #0000ff; }
 &lt;/style&gt;
 &lt;head/&gt;&lt;body&gt;
-&lt;p&gt;Pour plus d'information sur d&apos;Octave :&lt;/p&gt;
+&lt;p&gt;Pour plus d&apos;information sur d&apos;Octave :&lt;/p&gt;
 &lt;ul&gt;
 &lt;li&gt;Visiter &lt;a href=&quot;http://octave.org&quot;&gt;http://octave.org&lt;/a&gt;&lt;/li&gt;
 &lt;li&gt;Obtenir le documentation en ligne comme &lt;a href=&quot;http://www.gnu.org/software/octave/doc/interpreter/index.html&quot;&gt;html&lt;/a&gt;- or &lt;a href=&quot;http://www.gnu.org/software/octave/octave.pdf&quot;&gt;pdf&lt;/span&gt;&lt;/a&gt;-document&lt;/li&gt;
 
 ;/a&gt;-document&lt;/li&gt;
-&lt;li&gt;Ouvrir le navigateur de documentation de l'interface graphique d'Octave avec le menu d'aide.&lt;/li&gt;
+&lt;li&gt;Ouvrir le navigateur de documentation de l&apos;interface graphique d&apos;Octave avec le menu d&apos;aide.&lt;/li&gt;
 &lt;/ul&gt;
 &lt;/body&gt;&lt;/html&gt;</translation>
     </message>
     <message>
         <location line="+20"/>
         <source>Finish</source>
         <translation>Fin</translation>
     </message>
diff --git a/libgui/languages/nl_NL.ts b/libgui/languages/nl_NL.ts
--- a/libgui/languages/nl_NL.ts
+++ b/libgui/languages/nl_NL.ts
@@ -84,16 +84,89 @@
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1527"/>
         <source>copied selection to clipboard</source>
         <translation>selectie is naar klembord gekopieerd</translation>
     </message>
 </context>
 <context>
+    <name>QsciLexerBash</name>
+    <message>
+        <location filename="../../../qsci/qscilexerbash.cpp" line="+208"/>
+        <source>Default</source>
+        <translation type="unfinished">Standaardtekst</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Error</source>
+        <translation type="unfinished">Fout</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Comment</source>
+        <translation type="unfinished">Commentaar</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Number</source>
+        <translation type="unfinished">Getal</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Keyword</source>
+        <translation type="unfinished">Trefwoord</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Double-quoted string</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Single-quoted string</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Operator</source>
+        <translation type="unfinished">Operator</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Identifier</source>
+        <translation type="unfinished">Identifier</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Scalar</source>
+        <translation type="unfinished">Skalair getal</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Parameter expansion</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Backticks</source>
+        <translation type="unfinished">Backticks</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Here document delimiter</source>
+        <translation type="unfinished">Here document  scheidingsteken</translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Single-quoted here document</source>
+        <translation type="unfinished">Here document tussen aanhalingstekens</translation>
+    </message>
+</context>
+<context>
     <name>QsciLexerBatch</name>
     <message>
         <location filename="../../../qsci/qscilexerbatch.cpp" line="+179"/>
         <source>Default</source>
         <translation>Standaardtekst</translation>
     </message>
     <message>
         <location line="+3"/>
@@ -879,17 +952,17 @@ is reeds geopend in de editor</translati
         <source>&amp;Find and Replace</source>
         <translation>Zoek en Vervang</translation>
     </message>
     <message>
         <location line="-584"/>
         <source>Could not open file
 %1
 for read: %2.</source>
-        <translation >Kon bestand
+        <translation>Kon bestand
 %1
 niet openen om in te lezen: %2.</translation>
     </message>
     <message>
         <location line="+21"/>
         <source>File
 %1
 does not exist. Do you want to create it?</source>
@@ -1520,22 +1593,22 @@ is gewist of hernoemd. Wil je het nu ops
         <location line="+1"/>
         <source>Create script</source>
         <translation>Nieuw script</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation>Werkruimte laden</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation>Over Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation>Bestand</translation>
@@ -1591,23 +1664,33 @@ is gewist of hernoemd. Wil je het nu ops
         <translation>Kopiëren</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation>Plakken</translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation>Bewaar werkruimte als ...</translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation>Octave Release Notes</translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Stel werkmap in</translation>
     </message>
@@ -2287,17 +2370,17 @@ is gewist of hernoemd. Wil je het nu ops
     <message>
         <location line="+14"/>
         <source>Welcome to Octave!</source>
         <translation>Welkom bij Octave!</translation>
     </message>
     <message>
         <location line="+5"/>
         <source>You seem to be using the Octave graphical interface for the first  time on this computer.  Click &apos;Finish&apos; to write a configuration file  and launch Octave GUI.</source>
-        <translation>Kennelijk gebruik je Octave's grafische interface voor het eerst. Klik op Finish, dan wordt er een standaard configuratiebestand aangemaakt en daarna de Octave GUI gestart.</translation>
+        <translation>Kennelijk gebruik je Octave&apos;s grafische interface voor het eerst. Klik op Finish, dan wordt er een standaard configuratiebestand aangemaakt en daarna de Octave GUI gestart.</translation>
     </message>
     <message>
         <location line="+3"/>
         <source>The configuration file is stored in %1. If that file exists, you will not see this dialog when Octave starts again.</source>
         <translation>Het configuratiebestand is opgeslagen als %1. Als dat bestand bestaat zal Octave bij een volgende start dee dialoog overslaan.</translation>
     </message>
     <message>
         <location line="+23"/>
diff --git a/libgui/languages/pt_BR.ts b/libgui/languages/pt_BR.ts
--- a/libgui/languages/pt_BR.ts
+++ b/libgui/languages/pt_BR.ts
@@ -916,22 +916,22 @@ para:</translation>
         <location line="+1"/>
         <source>Create script</source>
         <translation>Criar script</translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation>Carregar ambiente de trabalho</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation>Sobre o Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation>Arquivo</translation>
@@ -987,23 +987,33 @@ para:</translation>
         <translation>Copiar</translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation>Colar</translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation>Salvar Ambiente de Trabalho como</translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation>Alterar diretório de trabalho</translation>
     </message>
diff --git a/libgui/languages/ru_RU.ts b/libgui/languages/ru_RU.ts
--- a/libgui/languages/ru_RU.ts
+++ b/libgui/languages/ru_RU.ts
@@ -905,22 +905,22 @@ has been deleted or renamed. Do you want
         <location line="+1"/>
         <source>Create script</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation type="unfinished">Загрузить область переменных</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation type="unfinished">Об Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation type="unfinished">&amp;Файл</translation>
@@ -976,23 +976,33 @@ has been deleted or renamed. Do you want
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation type="unfinished"></translation>
     </message>
diff --git a/libgui/languages/uk_UA.ts b/libgui/languages/uk_UA.ts
--- a/libgui/languages/uk_UA.ts
+++ b/libgui/languages/uk_UA.ts
@@ -84,16 +84,89 @@
     <name>QWinTerminalImpl</name>
     <message>
         <location filename="../qterminal/libqterminal/win32/QWinTerminalImpl.cpp" line="+1527"/>
         <source>copied selection to clipboard</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
+    <name>QsciLexerBash</name>
+    <message>
+        <location filename="../../../qsci/qscilexerbash.cpp" line="+208"/>
+        <source>Default</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Error</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Comment</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Number</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Keyword</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Double-quoted string</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Single-quoted string</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Operator</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Identifier</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Scalar</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Parameter expansion</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Backticks</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Here document delimiter</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+3"/>
+        <source>Single-quoted here document</source>
+        <translation type="unfinished"></translation>
+    </message>
+</context>
+<context>
     <name>QsciLexerBatch</name>
     <message>
         <location filename="../../../qsci/qscilexerbatch.cpp" line="+179"/>
         <source>Default</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+3"/>
@@ -1498,22 +1571,22 @@ has been deleted or renamed. Do you want
         <location line="+1"/>
         <source>Create script</source>
         <translation type="unfinished"></translation>
     </message>
 </context>
 <context>
     <name>main_window</name>
     <message>
-        <location filename="../src/main-window.cc" line="+162"/>
+        <location filename="../src/main-window.cc" line="+163"/>
         <source>Load Workspace</source>
         <translation type="unfinished">Завантажити область змінних</translation>
     </message>
     <message>
-        <location line="+420"/>
+        <location line="+435"/>
         <location line="+847"/>
         <source>About Octave</source>
         <translation type="unfinished">Про Octave</translation>
     </message>
     <message>
         <location line="-369"/>
         <source>&amp;File</source>
         <translation type="unfinished">&amp;Файл</translation>
@@ -1569,23 +1642,33 @@ has been deleted or renamed. Do you want
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+6"/>
         <source>Paste</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-1011"/>
-        <location line="+931"/>
+        <location line="-1026"/>
+        <location line="+946"/>
         <source>Save Workspace As</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
-        <location line="-809"/>
+        <location line="-826"/>
+        <source>The release notes file &apos;%1&apos; is empty.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+4"/>
+        <source>The release notes file &apos;%1&apos; cannot be read.</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <location line="+13"/>
         <source>Octave Release Notes</source>
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location line="+157"/>
         <source>Set working directory</source>
         <translation type="unfinished"></translation>
     </message>
